openbas-api/src/main/java/io/openbas/rest/tag_rule/TagRuleApi.java

```diff
@@ -36,14 +44,18 @@ public TagRuleApi(TagRuleService tagRuleService, TagRuleMapper tagRuleMapper) {
 
   @LogExecutionTime
   @GetMapping(TagRuleApi.TAG_RULE_URI + "/{tagRuleId}")
-  @Operation(summary = "Get TagRule by Id")
-  public TagRuleOutput findTagRule(@PathVariable @NotBlank final String tagRuleId) {
+  @Operation(description = "Get TagRule by Id", summary = "Get TagRule")
+  @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The tag rule")})
```

Comment:
Sometime, we talk about TagRule and other "tag rule". Should we uniformize this?

Reply:
Sure, I'll be trying to check if everything is fine